### ISSUE TITLE
image/video support?

### ISSUE THREAD

**Issue by vasnt** [Shaunwei/RealChar#70]
it would be great, if it has an eye(to see) so whatever i am seeing in physical world, i can ask my related queries to Realchar.

**Comment by Shaunwei** [comment]
yes. we need make AI companion to listen and understand first. Then see and predict

**Comment by Shaunwei** [comment]
move into backlog
